frontend/server/src/main/java/com/amazonaws/ml/mms/util/messages/AbstractRequest.java

```diff
@@ -1,6 +1,6 @@
 package com.amazonaws.ml.mms.util.messages;
 
-public abstract class AbstractRequest {
+public class AbstractRequest {
```

Comment:
It's better either keep it as abstract, or rename to BaseModelRequest.

Reply:
Shall change the base class.